staging/src/k8s.io/component-base/OWNERS

```diff
@@ -1,12 +1,14 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
+- sig-architecture-leads
 - mtaufen
 - stealthybox
 - luxas
 - sttts
 - lavalamp
 reviewers:
+- sig-architecture-pr-reviews
```

Comment:
you can drop this line

Reply:
double-checking; with @neolit123's suggestion this be the only reviewer line, do we still want to drop?

Reply:
I guess the bot is complaining about org membership for this identity... alternative?

Reply:
i know that there is a bug in test-infra owner assignment code / bots, where if there are no listed reviewers it would not take people from the list of approvers, but rather take owners from the parent OWNERS; in this case that would be:
https://github.com/kubernetes/kubernetes/blob/master/staging/OWNERS

Reply:
Ok, went with sig-architecture-approvers for both of these for now.